dcos/marathon.py

```diff
@@ -54,7 +54,11 @@ def _to_error(response):
     if isinstance(response, Error):
         return DefaultError(_default_marathon_error(response.error()))
 
-    message = response.json().get('message')
+    try:
+        message = response.json().get('message')
+    except ValueError as e:
```

Comment:
That's not actually what the error is.  The JSON error you're surfacing is the exception in response.json() where requests finds that it can't parse the response as JSON.  It's only a coincidence that this has something to do with the JSON-formatting of that command line argument.